content.php

```diff
@@ -11,6 +11,17 @@
 
 <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
 
+	<?php
+	/**
+	 * primer_before_content hook.
+	 *
+	 * @hooked primer_woo_shop_message - 10
```

Comment:
@EvanHerman this is not a docblock we are using anywhere else in the codebase, not sure it really scales either since hooks can often have lots of callbacks.